azurerm/internal/services/apimanagement/api_management_resource.go

```diff
@@ -294,6 +303,52 @@ func resourceArmApiManagementService() *schema.Resource {
 							Optional: true,
 							Default:  false,
 						},
+
+						"enable_tls_ecdhe_ecdsa_with_aes256_cbc_sha_ciphers": {
```

Comment:
Could we put enabled on the end of all these properties? this is how we name booleans now:
```suggestion
						"tls_ecdhe_ecdsa_with_aes256_cbc_sha_ciphers_enabled": {
```

Reply:
Done. Note: I renamed also the one cipher that was already there (`enable_triple_des_ciphers`) to have at least all cipher flags consistent. Technically this is a breaking change then.